### ISSUE TITLE
`mvn package` fails some tests

### ISSUE THREAD

**Issue by dariober** [biojava/biojava-legacy#37]
It appears some tests fail when building the package with maven:

```
cd biojava-legacy-biojava-legacy-1.9.2
mvn package
....
....
Results :

Failed tests: 

  GenbankRichSequenceDBTest.testeFetch:24 Unexpected Bioexception: org.biojava.bio.BioException: Failed to read Genbank sequence

  GenpeptRichSequenceDBTest.testeFetch:24 Unexpected BioException: org.biojava.bio.BioException: Failed to read Genbank sequence
...
...
```

See also http://mailman.open-bio.org/pipermail/biojava-l/2016-December/011509.html

Thank you!

**Comment by heuermh** [comment]
This may have been fixed by #34.  Can you pull from master and confirm?

**Comment by dariober** [comment]
That was quick! I confirm it works now. 

Many thanks! Dario
